flixel/input/mouse/FlxMouse.hx

```diff
@@ -397,8 +397,8 @@ class FlxMouse extends FlxPointer implements IFlxInputManager
 		//actually position the flixel mouse cursor graphic
 		if (visible)
 		{
-			cursorContainer.x = FlxG.game.mouseX;
-			cursorContainer.y = FlxG.game.mouseY;
+			cursorContainer.x = this.screenX;
```

Comment:
Evidently not as safe as I originally thought. It fails when using "normal" scale modes.
I'm working on a fix.